plot-compare.R

```diff
@@ -0,0 +1,21 @@
+library(ggplot2) ## plotting! may not be needed.
+library(pheatmap) ## for heatmap generation
+library(ggplotify) ## convert pheatmap to ggplot2 compatibility
+library(viridis) ## color palette stuff.
+
+args <- commandArgs(trailingOnly=TRUE)
+
+cat(args)
+
+mat_csv <- args[1]
+out_fig <- args[2]
+
+# read matrix CSV file, as output by sourmash compare --csv
+sourmash_comp_matrix <- read.csv(mat_csv)
```

Comment:
```suggestion
sourmash_comp_matrix <- read.csv(mat_csv)
colnames(sourmash_comp_matrix) <- gsub("..complete.sequence", "", colnames(sourmash_comp_matrix))
```
will at least shorten the labels so they...do less.